codespell_lib/data/dictionary.txt

```diff
@@ -9951,6 +9999,7 @@ opthalmology->ophthalmology
 opthamologist->ophthalmologist
 optiional->optional
 optimasation->optimization
+optimation->optimisation
```

Comment:
should probably add (and prefer) `optimization`

Reply:
Done